data_engineering/pyspark_tasks/task_1.py

```diff
@@ -0,0 +1,47 @@
+"""This module saves top 100 films according to IMDB ratings(all time,60th,last 10 years)"""
+from pyspark import SparkConf
+from pyspark.sql import SparkSession
+
+
+def get_best_alltime(basics_df, ratings_df):
+    return basics_df.join(ratings_df, basics_df['tconst'] == ratings_df['tconst'], 'inner') \
```

Comment:
for long lines of code with method chaining, it's better to use parentheses `(...)`, instead of  slash `\`, since it's "more"  python-way
e.g.
use:
```python
var = (some().very_long().chained()
       .method_line().sort())
```
instead of:
```python
var = some().very_long().chained() \
    .method_line().sort()
```